themes/src/main/resources/theme/keycloak-preview/account/resources/README.md

```diff
@@ -0,0 +1,48 @@
+New Account Console
+========
+
+The new Account Console uses PatternFly 4 and React.
+
+Here is what you need to know in order to do development on the new console.
+
+
+Building
+--------
+First, you must install node and npm.
+
+To build and lint the application:
+
+`npm run install`
```

Comment:
@ssilvert I can be wrong here, but I believe the correct would be `npm install`. Because `npm run install` will attempt to run a script and gives me this error:

```
~/g/k/t/s/m/r/t/k/a/resources ❯❯❯ npm run install                                                                                                            kc9659-readme ◼
npm ERR! missing script: install

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/abstractj/.npm/_logs/2019-03-19T18_19_31_731Z-debug.log
```

Reply:
You are right.  I will fix.